src/heroku.ts

```diff
@@ -0,0 +1,70 @@
+import Heroku from 'heroku-client';
+import { retry } from '@lifeomic/attempt';
+
+interface HerokuIntegrationConfig {
+  apiKey: string;
+}
+
+let heroku: Heroku = undefined;
```

Comment:
Module-scoped state has caused much pain since the lifecycle of a module's memory is difficult to control. I recommend against it.

Reply:
I changed the implementation in #2 - I think it's better. Have a look and let me know what you think.